src/sentry/api/helpers/mobile.py

```diff
@@ -0,0 +1,10 @@
+from sentry.api.helpers.android_models import ANDROID_MODELS
```

Comment:
With how giant this file is, do we know if there's any performance impact? Like I wonder if this is faster as a giant python file, or a CSV that we read in

Reply:
So I tested in pytest by wrapping the request in `timeit` and took the average of 500 `/tags` responses.
On this branch with the new `readable` flag, the average was 0.0835806546400001s
On this branch without the `readable` flag, the average was 0.08407600666000004s
And on `master` without any changes, the average was 0.08446629567999978s

Seems like there isn't any noticable performance impact? The difference between all test cases seem small enough that they could just be random.